common/static/common/js/spec/discussion/view/discussion_topic_menu_view_spec.js

```diff
@@ -45,7 +50,18 @@
                                     'sort_key': null,
                                     'is_cohorted': false,
                                     'id': 'c49f0dfb8fc94c9c8d9999cc95190c56'
+                                },
+                                'Very long category name': {
```

Comment:
It's just long enough to trigger the bug. The name that gets fed into `fitName` is 'Basic Question Types / Very long category name', which then gets shortened to '.../ Very long category name', which is just over the 26 characters, causing the infinite loop. It was trying to shorten 'Very long category name' instead of '.../ Very long category name'.